docs/getting_started.md

```diff
@@ -0,0 +1,46 @@
+## Getting started
+
+you can get a basic react system up and running extremely quickly by following these steps.
+
+1. Create an NPM account, if you don't have one and get an NPM token.
+
+2. Export your NPM token to the environment variable NPM_TOKEN.
```

Comment:
For a public consumer, you shouldn't need this. Only require a token for publishing or when pulling from a private NPM registry, which we aren't.

Reply:
steps removed